known-limitations.md

```diff
@@ -195,3 +195,11 @@ Given a query like `SELECT * FROM foo WHERE a > 1 OR b > 2`, even if there are a
 ## Privileges for `DELETE` and `UPDATE`
 
 Every [`DELETE`](delete.html) or [`UPDATE`](update.html) statement constructs a `SELECT` statement, even when no `WHERE` clause is involved. As a result, the user executing `DELETE` or `UPDATE` requires both the `DELETE` and `SELECT` or `DELETE` and `UPDATE` [privileges](privileges.html) on the table.
+
+## Background increase in amount of data
+
+The timeseries data used to power the graphs in the admin UI is stored within the cluster and accumulates for 30 days before it starts getting truncated. As a result, for the first 30 days or so of a cluster's life you will see a steady increase in disk usage and the number of ranges in the cluster even if you aren't writing data to it yourself. As of the 1.0 release, there is not yet a way to delete the timeseries data sooner than after 30 days, but if you want to save data point less frequently you can start each node with the `COCKROACH_METRICS_SAMPLE_INTERVAL` environment variable set higher than its default of `10s`. For example, you could set it to `1m` to only collect data every 1 minute, which would result in storing 6x less timeseries data than the default setting.
```

Comment:
Btw, this feels like feels like it does belong in know limitations, if we plan to make it possible to change the truncation period. If so, perhaps we should title it something like `Truncation period for timeseries data`, since the inability to change that period is the limitation?